contracts/Migrations.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.8.0;
+
+contract Migrations {
```

Comment:
Solidity noob question: what is this for?

Reply:
It's auto-generated by Truffle (https://github.com/lazyledger/evm-light-client/pull/1#discussion_r470651012). It's used for deploying new versions of a contract. It might also be used for deploying the initial version, but I'm not sure. To this day I don't know if it's actually a required part of truffle projects or if it can be removed, but I can look into removing it in a future PR. Docs: https://www.trufflesuite.com/docs/truffle/getting-started/running-migrations

Reply:
Actually, removed the migrations files in 115be04.